pkg/apis/rollouts/v1alpha1/types.go

```diff
@@ -989,6 +989,8 @@ const (
 	RolloutPaused RolloutConditionType = "Paused"
 	// RolloutCompleted means that rollout is in a completed state. It is still progressing at this point.
 	RolloutCompleted RolloutConditionType = "Completed"
+	// HealthyAndCompleted means that rollout is in a completed state and is healthy.
+	HealthyAndCompleted RolloutConditionType = "HealthyAndCompleted"
```

Comment:
I think `HealthyAndCompleted` is maybe the wrong way term this. While I understand that `HealthyAndCompleted` means the rollout is fully upgraded and all pods are available, I think this should just be called `Healthy`. The reason is: `status.phase` already terms this as `Healthy` (which indicates it is fully rolled out and all pods are available). By extension, Argo CD will also show this as `Healthy`. And so we would be introducing a new word for an existing concept. I think we should just refer to this across the board as `Healthy` and avoid introducing a new term.

We would indicate through explicit comments and docs that `Healthy` *implies* `Completed`

Reply:
Yup totally agree. It helped during the conversion process but yea no need to introduce a new term.